tests/wpunit/UserTest.php

```diff
@@ -0,0 +1,27 @@
+<?php 
+
+// class UserTest extends \Codeception\TestCase\WPTestCase
```

Comment:
I'll set that up as well. I didn't include at first because it needs it's own DB.  Apparently WPUnit loads, installs and configures fresh WordPress installation before each test method runs. I'll see what I can do.